README.md

```diff
@@ -108,21 +108,38 @@ optional arguments:
 
 ### Graphic
 
-The `graphic` sub-command will create either a comprehensive diagram showing ontology modules together with classes, object properties and "Things" together with the path of imports, or (if the 'wee' option is selected) a simple diagram of ontology modules and the import hierarchy.  Graphics are exported both as ```png``` files and also as a ```dot``` file.  This ```dot``` file can be used with Graphviz or with web tools such as [Model Viewer](http://www.semantechs.co.uk/model-viewer)
+The `graphic` sub-command will create either 
+* a comprehensive diagram showing ontology modules together with classes, object properties and "Things"
+  together with the path of imports, or (if the 'wee' option is selected) a simple diagram of ontology modules and
+  the import hierarchy, or
+* a diagram of the use of classes and object and data properties in a triple store.
+    
```

Comment:
line breaks are a bit odd here. Shows up badly in a markdown viewer:

![image](https://user-images.githubusercontent.com/10777545/111216461-b64a3200-8591-11eb-9e64-37bb1c44412a.png)

Reply:
Perhaps just an artifact of browser sizing? I get this

![image](https://user-images.githubusercontent.com/44446537/111216927-5eb8c000-85ab-11eb-8320-7247f073db46.png)

Reply:
I use using an online markdown viewer, probably buggy.